src/pickup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PICKUP_H
+#define _PICKUP_H
+
+#include <unordered_map>
+
+#include "entity.h"
+
+namespace pickupNS {
+	const enum PICKUP_TYPE { MAGNET, INVERT };
```

Comment:
Also if we use enum here it kinda defeats the purpose of loading it in from a json file